adapters/rest-adapter-core/src/main/java/uk/gov/justice/services/adapter/rest/JsonSchemaValidationInterceptor.java

```diff
@@ -38,13 +43,15 @@ public Object aroundReadFrom(final ReaderInterceptorContext context) throws IOEx
         if (context.getMediaType().getSubtype().endsWith(JSON_MEDIA_TYPE_FORMAT)) {
             final String charset = extractCharset(context);
             final String payload = IOUtils.toString(context.getInputStream(), charset);
-
+            final String name = extractName(context);
             try {
-                validator.validate(payload, extractName(context));
+                validator.validate(payload, name);
             } catch (ValidationException ex) {
-                throw new BadRequestException(ex.getMessage(), ex);
+                LOGGER.debug(String.format("JSON schema validation has failed on %s for : %s ",
```

Comment:
I prefer the message structure used in the JMS code "JSON schema validation has failed for %s due to %s"